e-commerce/src/__tests__/Registration.test.tsx

```diff
@@ -0,0 +1,493 @@
+import { describe, it, expect } from 'vitest';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/store/store';
+import * as Pages from '../pages/pages';
+
+/* Registration page */
+describe('render Registration page', () => {
+  it('renders Title on Registration page', () => {
+    render(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={['/']}>
+          <Pages.Registration />
+        </MemoryRouter>
+      </Provider>
+    );
+
+    expect(screen.queryByText('Registration')).toBeInTheDocument();
+  });
+});
+
+/* Label */
+describe('checking input labels on Registration page', () => {
+  it('checking input label for email on Registration page', () => {
+    render(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={['/']}>
+          <Pages.Registration />
+        </MemoryRouter>
+      </Provider>
+    );
+
+    expect(screen.getByLabelText('Email*'));
```

Comment:
Здесь не нужен expect
```suggestion
    screen.getByLabelText('Email*');
```

Reply:
Пофиксили

Reply:
Там ниже еще много таких же